app/src/org/commcare/utils/CommCareExceptionHandler.java

```diff
@@ -57,4 +57,14 @@ private boolean warnUserAndExit(Throwable ex) {
         }
         return false;
     }
+
+    private static boolean causedByLocalizationException(Throwable ex) {
+        if (ex == null) {
+            return false;
+        }
+        if (ex instanceof NoLocalizedTextException) {
+            return true;
+        }
+        return causedByLocalizationException(ex.getCause());
```

Comment:
Not actually seeing how I can do that and get the identical result.. What would you change it to?